rotkehlchen/db/drivers/db_messages.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+enum DBConnectionType {
+  USER = 0;
+  TRANSIENT = 1;
+  GLOBAL = 2;
+}
+
+enum DBMethod {
+  INITIALIZE = 0;
+  OPEN_CURSOR = 1;
+  CLOSE_CURSOR = 2;
+  EXECUTE = 3;
+  EXECUTEMANY = 4;
+  EXECUTESCRIPT = 5;
+  ROLLBACK = 6;
+  COMMIT = 7;
+  CLOSE = 8;
+}
+
+message DBWriteResult {
+  int32 rowcount = 1;
+  uint64 lastrowid = 2;
+}
+
+message DBWriterError {
+  string message = 1;
+  optional string module = 2;
+  string class_name = 3;
+}
```

Comment:
Not sure about the errors... As you are writing them you are passing python errors around. It is okey if it is python processes but in the future we will have other type of errors... Don't you think it would be nice to have something that works for all the languages? In the client you can raise a `DBWrittingError` that includes the error type and the message... I guess the complexity and why you didn't do it is because you will need to change the handling of errors around the app. What kind of errors do we have? OperationalError, IntegrityError and then something unexpected right?